README.md

```diff
@@ -1,11 +1,133 @@
-<h1 align="center">firestore-meilisearch ⚠ WIP ⚠️</h1>
+<h1 align="center">⚠ WIP ⚠️</h1>
 
 <p align="center">
+  <img src="https://raw.githubusercontent.com/meilisearch/integration-guides/main/assets/logos/meilisearch_firebase.svg" alt="Meilisearch-Firestore" width="200" height="200" />
+</p>
+
+<h1 align="center">Meilisearch Firestore</h1>
+
+<h4 align="center">
+  <a href="https://github.com/meilisearch/meilisearch">Meilisearch</a> |
+  <a href="https://docs.meilisearch.com">Documentation</a> |
+  <a href="https://slack.meilisearch.com">Slack</a> |
+  <a href="https://roadmap.meilisearch.com/tabs/1-under-consideration">Roadmap</a> |
+  <a href="https://www.meilisearch.com">Website</a> |
+  <a href="https://docs.meilisearch.com/faq">FAQ</a>
+</h4>
+
+<p align="center">
+  <a href="https://packagist.org/packages/meilisearch/firestore-meilisearch"><img src="https://img.shields.io/packagist/v/meilisearch/firestore-meilisearch" alt="Latest Stable Version"></a>
```

Comment:
not sure firestore extension is in packagist

Reply:
We don't ...